rel-eng/compare-with-tags

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+#
+# Compares tito.props with koji and prints out differences in Github markup for
+# review and cleanup.
+#
+
+from configparser import ConfigParser
+from pathlib import Path
+from subprocess import check_output
+from typing import Set
+
+
+def sclize(package: str, scl: str) -> str:
+    if package.startswith(('rubygem-', 'nodejs-')):
+        return f"{scl}-{package}"
+    return package
+
+
+def get_koji_packages(tag: str) -> Set[str]:
+    command = ['koji', 'list-pkgs', f'--tag={tag}', '--quiet']
+    output = check_output(command, universal_newlines=True)
+    return {line.split(None, 1)[0] for line in output.splitlines()}
+
+
+def main(filename: Path) -> None:
+    cfg = ConfigParser()
+    cfg.read(filename)
```

Comment:
this silently fails if `failname` doesn't exist, which is rather weird. not that this should happen in our setup (unless Evgeni copies the file to the wrong folder for testing), but still weird

Reply:
Updated with code to verify it was read.